keep/iohandler/iohandler.py

```diff
@@ -72,10 +72,29 @@ def parse(self, string):
         #           first(split({{ foreach.value }},'a', 'b'))
 
         # first render everything using chevron
+
+        # if the string is a simple render, e.g. {{ value }}
+        if (
+            string.count("{{") == 1
+            and string.strip().startswith("{{")
+            and string.strip().endswith("}}")
+        ):
+            simple_render = True
+        else:
+            simple_render = False
```

Comment:
I think this is redundant

Reply:
I'll remove it but I'm 100% sure it'll break other alerts. anyway, I've removed most of the alerts so..